docs/09-Configuration reference/02-Custom Resource Definitions.md

```
@@ -2356,6 +2502,65 @@ GatewayHTTPAPI is the Schema for the HTTPAPIs API
+
+
+
+
+
+
+
+
+| Field | Description | Default | Validation |
+| --- | --- | --- | --- |
+| `stacks` _string array_ | Stacks on which the configuration is applied. Can contain `*` to<br />indicate a wildcard, following the same convention as Settings. |  |  |
+| `cluster` _[ClusterSpec](#clusterspec)_ | Cluster is the base Ledger v3 Cluster specification. Stack-specific<br />Settings and values owned by the Operator are applied on top of it. |  |  |
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)